src/scripts/mobileMenu.js

```diff
@@ -12,12 +12,19 @@
     modal.style.display = 'none';
   };
 
-  linkMenu.onclick = function () {
-    modal.style.display = 'none';
-  };
+  Array.from(linkItem).forEach(element => {
+    element.onclick = function () {
+      modal.style.display = 'none';
+    };
+  });
 
   window.onclick = function (event) {
-    if (event.current.target == modal) {
+    if (
+      !modal.contains(event.target) &&
+      event.target !== modal &&
+      event.target !== openMenu &&
+      !openMenu.contains(event.target)
+    ) {
       modal.style.display = 'none';
     }
```

Comment:
Apparently, lines 24 and 25 are already included in the lines before and after them